spring-xd-lang-detector/README.md

```diff
@@ -0,0 +1,104 @@
+Spring XD Language Detector Example
+===============================
+
+This is an example of a custom module that uses the ([langdetect](https://code.google.com/p/language-detection/)) API.
```

Comment:
Not sure those parens were meant to be around all links.

Should consider asciidoc if you've got some spare time, as this is what we use in many other places